src/commons/components/progress-bar/progress-bar.html

```diff
@@ -0,0 +1,24 @@
+<div class="progress-bar-component">
+	<div class="progress">
+		<div ng-repeat="progress in vm.progressData.data" class="progress-bar" role="progressbar" ng-style="vm.getStyle($index)">
+	    </div>
+	    <div class="progress-bar" role="progressbar" ng-style="{width: vm.remainingValue , background: white}">
```

Comment:
@kamleshverma1 As all other colors are configurable, can you make this "white" color also configurable?